### ISSUE TITLE
[bug] Pitch maintained when adjusting speed of video

### ISSUE THREAD

**Issue by chenlung** [paulpacifico/shutter-encoder#15]
1. Open video with PAL speedup.
2. In Advanced feature, Conform by speed 23.976.
3. Start function.

Expected result: Both video and audio are slowed to 23.976.
Actual result: Video is slowed to 23.976; audio is PAL-pitch and 23.976 tempo.

Using Shutter Encoder 14.6 - Mac 10.11.6.

**Comment by chenlung** [comment]
For some, this might be a feature; to others, it's a bug.

I don't know if there should be an additional option?

**Comment by paulpacifico** [comment]
This is not a bug, i decided to maintain audio pitch with conform by speed option.

**Comment by chenlung** [comment]
Okay.

Can we have an option to slow both video and audio down? I can personally only do this using Cinema Tools (Apple) on certain file types.

**Comment by paulpacifico** [comment]
Yes why not, i just need to think how to implement this, like a checkbox with "Maintain audio pitch"

**Comment by chenlung** [comment]
I think the default could be slowing both down, but you indeed have a checkbox option with 'Maintain audio pitch' (or 'Maintain pitch').

**Comment by paulpacifico** [comment]
For sure, it's a great idea.

**Comment by chenlung** [comment]
It might be ideal having an option to not resample the Hz if slowing or speeding video (like Cinema Tools).

48000 25fps > 46080 24fps / 46034 23.976

**Comment by paulpacifico** [comment]
After some research it seems the '**asetrate**' filter of FFmpeg can do that easily!

**Comment by chenlung** [comment]
It doesn't just apply to video, it would seem.

I tried changing a WAV from 25 to 23.976; whilst the tempo was adjusted, the pitch was maintained.
